docs/docs/interactive-pages.md

```diff
@@ -2,7 +2,147 @@
 title: Interactive pages
 ---
 
-This is a stub. Help our community expand it.
+One nice thing about using Gatsby for building websites vs. other tools is that itʼs easier to add interactivity to your pages. React.js was designed for
+Facebook.com and is used on many other world-class web applications.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Let's see how to add interactive elements to our pages. Let's start with a counter.
+
+We'll start by creating a new link to a page at `/counter`/ from our original
+`index.js` page component `<Link to="/counter/">Counter</Link>`.
+
+```jsx{13-15}
+import React from "react"
+import { Link } from "gatsby"
+
+export default () =>
+  <div style={{ color: `tomato` }}>
+    <h1>Hello Gatsby!</h1>
+    <p>What a world.</p>
+    <img src="https://source.unsplash.com/random/400x200" alt="" />
+    <br />
+    <div>
+      <Link to="/page-2/">Link</Link>
+    </div>
+    <div>
+      <Link to="/counter/">Counter</Link>
+    </div>
+  </div>
+```
+
+Add that link, click on it, and then we'll create a "Hello World" page component
+for `/counter/` as before. But instead of using the "functional component" form
```

Comment:
> as before

This is probably an accidental hanging reference to where this was sourced from?